Code/Framework/AzCore/AzCore/azcore_files.cmake

```diff
@@ -298,6 +298,8 @@ set(FILES
     Math/MathUtils.h
     Math/MathMatrixSerializer.h
     Math/MathMatrixSerializer.cpp
+    Math/MathStringConversions.h
+    Math/MathStringConversions.cpp
     Math/MathVectorSerializer.h
     Math/MathVectorSerializer.cpp
```

Comment:
nit: it seems slightly redundant to have 'Math' in the filename when they're in a 'Math' folder

Reply:
While I do agree, I favored following the existing convention here.